staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -209,7 +209,7 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	infos := []*resource.Info{}
-	err = r.Visit(func(info *resource.Info, err error) error {
+	_ = r.Visit(func(info *resource.Info, err error) error {
```

Comment:
aren't we supposed to handle that error? I think it was just forgotten.

Reply:
the return “err” is always nil, so is there nessary to handle it?

Reply:
ack

Reply:
Why not:
```
r.Visit(...
```
if we don't care about return value, don't assign it at all.

Reply:
> Why not:
> 
> ```
> r.Visit(...
> ```
> 
> if we don't care about return value, don't assign it at all.

ok, I will update the code.